packages/safe-tools-client/app/components/schedule-payment-form-action-card/index.gts

```diff
@@ -376,6 +383,53 @@ export default class SchedulePaymentFormActionCard extends Component<Signature>
     return state;
   });
 
+  @use requiredGasTokenToSchedulePayment = resource(() => {
+    const state: RequiredGasTokenToSchedulePaymentState = new TrackedObject({
+      isLoading: false,
+      isIndeterminate: false
```

Comment:
Can you explain how `isIndeterminate` is used and why is it needed? What is it that we can't determine?

Reply:
Yep we don't need this field. Removed